plugins/cleanpaste/trumbowyg.cleanpaste.js

```diff
@@ -63,7 +63,7 @@
         // Remove multiple spaces
         html.replace(/\s+/g, ' ');
 
-        // strip &nbsp; -cgCraft
+        // strip leading and trailing spaces
```

Comment:
trim spaces

Reply:
I'm happy with "trim spaces" if you prefer - but I'm sorry, I made this change on Github directly and I don't know how to edit it now! By all means, bounce the PR and make the change yourself...